socorro/external/rabbitmq/crashstorage.py

```diff
@@ -79,40 +63,18 @@ def __init__(self, config, namespace='', quit_check_callback=None):
             quit_check_callback=quit_check_callback
         )
 
-        self.config = config
-
         # Note: this may continue to grow if we aren't acking certain UUIDs.
         # We should find a way to time out UUIDs after a certain time.
         self.acknowledgement_token_cache = {}
         self.acknowledgment_queue = Queue()
 
         self.rabbitmq = config.rabbitmq_class(config)
-        self.transaction = config.transaction_executor_class(
-            config,
-            self.rabbitmq,
-            quit_check_callback=quit_check_callback
-        )
-
-        # cache this object so we don't have to remake it for every transaction
         self._basic_properties = pika.BasicProperties(
-            delivery_mode=2,  # make message persistent
+            # Make message persistent
+            delivery_mode=2,
         )
 
-        if config.throttle == 100:
-            self.dont_queue_this_crash = lambda: False
-        else:
-            self.dont_queue_this_crash = (
-                lambda: randint(1, 100) > config.throttle
-            )
-
     def save_raw_crash(self, raw_crash, dumps, crash_id):
-        if self.dont_queue_this_crash():
-            self.config.logger.info(
-                'Crash %s filtered out of RabbitMQ queue %s',
-                crash_id,
-                self.config.routing_key
-            )
-            return
         try:
             this_crash_should_be_queued = (
                 not self.config.filter_on_legacy_processing or
```

Comment:
I could probably nix "legacy_processing" check and related code, too, but I figured testing that is involved and this PR is a little out of control at this point.